daemon/algod/api/server/v1/handlers/handlers.go

```diff
@@ -92,6 +92,8 @@ func txEncode(tx transactions.Transaction, ad transactions.ApplyData) (v1.Transa
 		res = assetFreezeTxEncode(tx, ad)
 	case protocol.ApplicationCallTx:
 		res = applicationCallTxEncode(tx, ad)
+	case protocol.CompactCertTx:
```

Comment:
Mostly because, as far as I can tell, the e2e testing code uses the v1 API, so it was easiest to extend it that way.

Somewhere (I forget where, unfortunately) Rotem commented that the v2 API doesn't need to be explicitly extended to support this because it just sends back the entire msgpack-encoded transaction, so the v2 API client can get whatever information it wants directly.